backend/src/appointment/database/repo/subscriber.py

```diff
@@ -71,8 +72,32 @@ def update(db: Session, data: schemas.SubscriberIn, subscriber_id: int):
     return db_subscriber
 
 
+def disable(db: Session, subscriber: models.Subscriber):
+    """Disable a given subscriber"""
+    # mark subscriber deleted = disable them
+    subscriber.time_deleted = datetime.datetime.now(datetime.UTC)
+    # invalidate session
+    # subscriber.minimum_valid_iat_time = datetime.datetime.now(datetime.UTC)
+    db.add(subscriber)
+    db.commit()
+    db.refresh(subscriber)
+    return subscriber
+
+
+def enable(db: Session, subscriber: models.Subscriber):
+    """Enable a given subscriber"""
+    # mark subscriber deleted = disable them
+    subscriber.time_deleted = None
+    # refresh session
+    # subscriber.minimum_valid_iat_time = datetime.datetime.now(datetime.UTC)
+    db.add(subscriber)
+    db.commit()
+    db.refresh(subscriber)
+    return subscriber
+
+
 def delete(db: Session, subscriber: models.Subscriber):
-    """Delete a subscriber by subscriber id"""
```

Comment:
Can we swap this function to hard_delete to be explicit, I think it's only used in two places.

Reply:
Of course